tests/units/actions/test_action_base.py

```diff
@@ -7,19 +7,23 @@
 class FakeAction(ActionBase):
     ActionBase.__abstractmethods__ = set()
     __id__ = None
+
+    def __init__(self, name):
+        self.name = name
```

Comment:
Why is that if you already have the id? Besides, `ActionBase` doesn't have `name` in `__init__`.

Reply:
There was a `get_named_actions` method in the reference code for this class and i found it interesting for debugging pourposes. Then, i thought the name and the id were separete things, like one for the user and the other for the system. 

I had put the name attribute in `ActionBase` but i think it was lost when i changed to this branch.

Reply:
If I remember correctly, the `id` would also be defined by the user. So it ends up that `id` and `name` have the same purpose, making it redundant to have them. I think we can remove `name`.